README.md

```diff
@@ -1,2 +1,40 @@
-TODO
-### Che server
\ No newline at end of file
+[![Next build Status](https://github.com/eclipse-che/che-theia/actions/workflows/next-build.yml/badge.svg)]
+[![Release build Status](https://github.com/eclipse-che/che-theia/actions/workflows/release.yml/badge.svg)]
+
+# What is Che server
+Che Server is a core component of the [Eclipse Che](https://github.com/eclipse/che/). This component is responsibe for creation and managing of Che workspaces, but will some day be replaced by the [Dev Workspace Operator](https://github.com/devfile/devworkspace-operator). 
+
+# Project structure
+Che Server is primarily a Java web application deployed on a Apache Tomcat server in a container. 
+
+The primary root module ['pom.xml'](https://github.com/eclipse-che/che-server/tree/main/pom.xml) lists all the java submodules. 
+
+The ['dockerfiles'](https://github.com/eclipse-che/che-server/tree/main/dockerfiles) directory contains container image for Che Server, as well as additional images, such as container for PostgreSQL DB. 
```

Comment:
This description is weird, there are no container images, there are only dockerfiles to build the images.

Reply:
will fix